test/CMakeLists.txt

```diff
@@ -454,14 +454,14 @@ endif()
 # Track
 set(CELERITASTEST_PREFIX celeritas/track)
 celeritas_add_test(celeritas/track/Sim.test.cc ${_needs_geant4})
-celeritas_add_test(celeritas/track/TrackSort.test.cc GPU ${_needs_geant4})
+celeritas_add_test(celeritas/track/TrackSort.test.cc GPU ${_needs_geant4} ${_needs_geo})
```

Comment:
Is it okay for `celeritas_add_test` to get the 'same' argument several times (i.e. `DISABLE`)?